gcloud-java-dns/src/test/java/com/google/gcloud/dns/ChangeRequestTest.java

```diff
@@ -16,203 +16,132 @@
-    ChangeRequest empty = ChangeRequest.builder().build();
-    assertEquals(empty.hashCode(), ChangeRequest.builder().build().hashCode());
-  }
-
-  @Test
-  public void testToAndFromPb() {
-    assertEquals(CHANGE, ChangeRequest.fromPb(CHANGE.toPb()));
-    ChangeRequest partial = ChangeRequest.builder().build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-    partial = ChangeRequest.builder().id(ID).build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-    partial = ChangeRequest.builder().add(RECORD1).build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-    partial = ChangeRequest.builder().delete(RECORD1).build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-    partial = ChangeRequest.builder().additions(ADDITIONS).build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-    partial = ChangeRequest.builder().deletions(DELETIONS).build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-    partial = ChangeRequest.builder().startTimeMillis(START_TIME_MILLIS).build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-    partial = ChangeRequest.builder().status(STATUS).build();
-    assertEquals(partial, ChangeRequest.fromPb(partial.toPb()));
-  }
-
-  @Test
-  public void testToBuilder() {
-    assertEquals(CHANGE, CHANGE.toBuilder().build());
-    ChangeRequest partial = ChangeRequest.builder().build();
-    assertEquals(partial, partial.toBuilder().build());
-    partial = ChangeRequest.builder().id(ID).build();
-    assertEquals(partial, partial.toBuilder().build());
-    partial = ChangeRequest.builder().add(RECORD1).build();
-    assertEquals(partial, partial.toBuilder().build());
-    partial = ChangeRequest.builder().delete(RECORD1).build();
-    assertEquals(partial, partial.toBuilder().build());
-    partial = ChangeRequest.builder().additions(ADDITIONS).build();
-    assertEquals(partial, partial.toBuilder().build());
-    partial = ChangeRequest.builder().deletions(DELETIONS).build();
-    assertEquals(partial, partial.toBuilder().build());
-    partial = ChangeRequest.builder().startTimeMillis(START_TIME_MILLIS).build();
-    assertEquals(partial, partial.toBuilder().build());
-    partial = ChangeRequest.builder().status(STATUS).build();
-    assertEquals(partial, partial.toBuilder().build());
-  }
-
-  @Test
-  public void testClearAdditions() {
-    ChangeRequest clone = CHANGE.toBuilder().clearAdditions().build();
-    assertTrue(clone.additions().isEmpty());
-    assertFalse(clone.deletions().isEmpty());
+  private static final DnsOptions OPTIONS = createStrictMock(DnsOptions.class);
+
+  private Dns dns;
+  private ChangeRequest changeRequest;
+  private ChangeRequest changeRequestPartial;
+
+  @Before
+  public void setUp() throws Exception {
+    dns = createStrictMock(Dns.class);
+    expect(dns.options()).andReturn(OPTIONS);
```

Comment:
Here and in the following you could use [times(int count)](http://easymock.org/api/org/easymock/IExpectationSetters.html#times-int-)